tooling-cpu-load/src/main/java/org/jboss/eap/qe/microprofile/tooling/cpu/load/utils/JavaProcessBuilder.java

```diff
@@ -0,0 +1,98 @@
+package org.jboss.eap.qe.microprofile.tooling.cpu.load.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Helper builder class for starting Java process.
+ */
+public class JavaProcessBuilder {
```

Comment:
I suggest to make API of this builder fluent - e.g. each method setting a value in builder should return its instance. WDYT?

Reply:
+1

Reply:
fixed